packages/terra-dialog/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "terra-dialog",
   "main": "lib/Dialog.js",
   "version": "0.0.0",
-  "description": "{insert description}",
+  "description": "Wrapper for terra-content-container intended to be consumed from within a modal or popup.",
```

Comment:
Let's update this with the description update to packages/terra-dialog/docs/README.md